pkgs/applications/window-managers/sway/ws.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitLab, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "swayws";
+  version = "unstable-2022-03-10";
+
+  src = fetchFromGitLab {
+    owner = "w0lff";
+    repo = pname;
+    rev = "514f3c664439cf2c11bb9096c7e1d3b8c0b898a2";
+    sha256 = "sha256-vUnbn79v08riYLMBI8BxeBPpe/pHOWlraG7QAaohw3s=";
+  };
+
+  cargoSha256 = "sha256-PvKpcTewajvbzUHPssBahWVcAQB3V/aMmOJ/wA0Nrv4=";
+
+  nativeBuildInputs = [];
+  buildInputs = [ ];
```

Comment:
You can remove these empty arrays, they're the default.

Reply:
Thanks for the review! I removed them now.